static/js/all-alerts.js

```diff
@@ -53,6 +53,25 @@ function getAllAlerts(){
         displayAllAlerts(res.alerts);
     })
 }
+// Custom cell renderer for State field
+function stateCellRenderer(params) {
+    let state = params.value;
+    let color;
+    switch (state) {
+        case 'Normal':
+            color = '#53DB6E';
```

Comment:
dont use hard coded number, use color names, also dark/light theme might not work with this

Reply:
they are the colors for state of the Alert ...will that not be same throughout for light as well as dark theme?